### DISCUSSION TITLE
阻止系统休眠

### DISCUSSION THREAD

**Discussion by Lockik** [gkd-kit/gkd#336] [Ideas]
这个版本会阻止系统休眠  耗电过快![Screenshot_20231130_150548_com.huawei.systemmanager.jpg](https://github.com/gkd-kit/gkd/assets/118150761/46ec2332-5946-4fe0-88e0-87f8f180168d)

**Comment by lisonge** [comment]
gkd 哪个版本不会阻止系统休眠 ？

**Reply by Lockik** [reply]
这样耗电有点快   能不能优化一下

**Reply by lisonge** [reply]
你先回答我的问题再说，否则建议你觉得耗电可以直接卸载

**Reply by Lockik** [reply]
我怎么回答  我就用了这一个版本

**Reply by lisonge** [reply]
抱歉我不知道为什么会阻止你的系统休眠，所以我不知道怎么优化，你可以暂时卸载等待后续版本的调整
